src/utils/wallet.js

```diff
@@ -338,19 +363,20 @@ const wallet = {
     // When we start a wallet from the locked screen, we need to unlock it in the storage
     oldWalletUtil.unlock();
  
-    // This check is important to set the correct network on storage and redux
-    const data = await version.checkApiVersion();
-
+    const network = config.getNetwork();
     const dataToken = tokens.getTokens();
 
     // Before cleaning loaded data we must save in redux what we have of tokens in localStorage
     store.dispatch(reloadData({ tokens: dataToken }));
 
+    // We are offline, the connection object is yet to be created
+    store.dispatch(isOnlineUpdate({ isOnline: false }));
+
     // Fetch metadata of all tokens registered
-    this.fetchTokensMetadata(dataToken.map((token) => token.uid), data.network);
+    this.fetchTokensMetadata(dataToken.map((token) => token.uid), network.name);
 
     const uniqueDeviceId = walletHelpers.getUniqueId();
-    const featureFlags = new FeatureFlags(uniqueDeviceId, data.network);
+    const featureFlags = new FeatureFlags(uniqueDeviceId, network.name);
```

Comment:
If I try to get a feature flag with network `privatenet`, then unleash will return that it does not exist and we will always use the old facade?

Reply:
That is correct